app/src/main/java/org/jellyfin/androidtv/auth/AuthenticationRepository.kt

```diff
@@ -103,7 +103,12 @@ class AuthenticationRepositoryImpl(
 			}
 
 			authenticationStore.getUser(server.id, user.id)?.let { storedUser ->
-				authenticationStore.putUser(server.id, user.id, storedUser.copy(lastUsed = Date().time))
+				authenticationStore.putUser(server.id, user.id, storedUser.copy(
```

Comment:
The user object send to this function always comes from our own store or it's just been made in the login function. Updating the data doesn't do anything.

Reply:
This was required in my testing because a stored user would never be updated if an auth token had been stored.

Reply:
I think that's because it is called by the authenticateUser function which is called with the changed user data you set in the `getServerUsers` function, which shouldn't really do that.

Reply:
The big question I still have here is if we should not be updating the user in the authentication store here, then where should that be happening?

Reply:
I think directly in in the place the changes to the user object are made. So the `getServerUsers` functions I guess.